src/main/proto/ga4gh/rna_quantification_service.proto

```diff
@@ -7,44 +7,55 @@ import "ga4gh/rna_quantification.proto";
 service RnaQuantificationService {
 // Gets a list of 'RnaQuantification' matching the search criteria.
 
-// 'POST /rnaquantification/search' must accept JSON version of
+// 'POST /rnaquantifications/search' must accept JSON version of
 // 'SearchRnaQuantificationRequest' as the post body and will return a JSON
 // version of 'SearchRnaQuantificationResponse'.
 rpc SearchRnaQuantifications(SearchRnaQuantificationsRequest)
       returns (SearchRnaQuantificationsResponse);
 
 
 // Gets a `RnaQuantification` by ID.
-// `GET /rnaquantification/{id}` will return a JSON version of
+// `GET /rnaquantifications/{id}` will return a JSON version of
 // `RnaQuantification`.
 rpc GetRnaQuantification(GetRnaQuantificationRequest)
       returns (RnaQuantification);
 
 // Gets a list of 'ExpressionLevel' matching the search criteria.
 
-// 'POST /expressionlevel/search' must accept JSON version of
+// 'POST /expressionlevels/search' must accept JSON version of
 // 'SearchExpressionLevelRequest' as the post body and will return a JSON
 // version of 'SearchExpressionLevelResponse'.
 rpc SearchExpressionLevels(SearchExpressionLevelsRequest)
       returns (SearchExpressionLevelsResponse);
 
+// Gets a `ExpressionLevel` by ID.
+// `GET /expressionlevels/{id}` will return a JSON version of
+// `ExpressionLevel`.
+rpc GetExpressionLevel(GetExpressionLevelRequest)
+      returns (ExpressionLevel);
 
-// Gets a list of 'QuantificationGroup' matching the search criteria.
+// Gets a list of 'ExpressionLevelGroup' matching the search criteria.
 
-// 'POST /quantificationgroup/search' must accept JSON version of
-// 'SearchQuantificationGroupRequest' as the post body and will return a JSON
-// version of 'SearchQuantificationGroupResponse'.
-rpc SearchQuantificationGroups(SearchQuantificationGroupsRequest)
-      returns (SearchQuantificationGroupsResponse);
+// 'POST /expressionLevelgroups/search' must accept JSON version of
+// 'SearchExpressionLevelGroupRequest' as the post body and will return a JSON
+// version of 'SearchExpressionLevelGroupResponse'.
+rpc SearchExpressionLevelGroups(SearchExpressionLevelGroupsRequest)
+      returns (SearchExpressionLevelGroupsResponse);
 }
 
-// ****************  /rnaquantification/search  *******************
+// Gets a `ExpressionLevelGroup` by ID.
+// `GET /expressionlevelgroups/{id}` will return a JSON version of
+// `ExpressionLevelGroup`.
+rpc GetExpressionLevelGroup(GetExpressionLevelGroupRequest)
+      returns (ExpressionLevelGroup);
 
-// This request maps to the body of 'POST /rnaquantification/search'
+// ****************  /rnaquantifications/search  *******************
+
+// This request maps to the body of 'POST /rnaquantifications/search'
 // as JSON.
 message SearchRnaQuantificationsRequest {
   // If present, return only Rna Quantifications which belong to this set.
-  string rna_quantification_id = 1;
+  string rna_quantification_set_id = 1;
```

Comment:
I believe this is pointing to a new message type, `QuantificationSetId`, which must be defined as well and I presume offer its own API methods (search sets, get sets).